backend/config/environments/development.rb

```diff
@@ -40,17 +40,7 @@
   # config.action_view.raise_on_missing_translations = true
 
   config.action_mailer.default_url_options = {host: "localhost", port: 3000}
-  config.action_mailer.delivery_method = :smtp
-
-  ActionMailer::Base.smtp_settings = {
-    address: "smtp.sendgrid.net",
-    port: "587",
-    authentication: :plain,
-    user_name: ENV["SENDGRID_USERNAME"],
-    password: ENV["SENDGRID_PASSWORD"],
-    domain: "localhost",
-    enable_starttls_auto: true
-  }
+  config.action_mailer.delivery_method = :sparkpost
```

Comment:
Do you think it is good idea to use for example `letter_opener` and not use smtp server at develop?

Reply:
OK, agree!